.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -63,6 +63,15 @@ jobs:
         env:
           TARGET_BRANCH: ${{ inputs.target-branch }}
 
+      - name: get-kata-tarball
+        uses: actions/download-artifact@v3
+        with:
+          name: kata-static-tarball-amd64${{ inputs.tarball-suffix }}
+          path: kata-artifacts
+
+      - name: Install kata
+        run: bash tests/integration/kubernetes/gha-run.sh install-kata kata-artifacts
```

Comment:
I believe the right approach here is to add genpolicy to kata-deploy, otherwise this step would conflict with the Deploy Kata step.